### ISSUE TITLE
1.1.341: False positive involving ParamSpec

### ISSUE THREAD

**Issue by rsokl** [microsoft/pyright#6759]
```python
def _flat_call(x: Iterable[Callable[P, Any]]) -> Callable[P, None]:
    def f(*args: P.args, **kwargs: P.kwargs) -> None:
        for fn in x:
            fn(*args, **kwargs)  # error: Arguments for ParamSpec "P@_flat_call" are missing (reportGeneralTypeIssues)

    return f
```

**Comment by erictraut** [comment]
Thanks for the bug report. This will be addressed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.342, which I just published. It will also be included in a future release of pylance.
